airflow/cli/commands/db.py

```diff
@@ -314,7 +314,7 @@ def __str__(self):
         return str(sorted(config_dict))
 
 
-@db.command('cleanup')
+@db.command('clean')
```

Comment:
It looks like the current CLI defines `clean` instead of `cleanup`

Reply:
Thank you.